actpool/actpool.go

```diff
@@ -240,11 +240,33 @@ func (ap *actPool) GetPendingNonce(addr string) (uint64, error) {
 func (ap *actPool) GetUnconfirmedActs(addr string) []action.SealedEnvelope {
 	ap.mutex.RLock()
 	defer ap.mutex.RUnlock()
-
+	var ret []action.SealedEnvelope
 	if queue, ok := ap.accountActs[addr]; ok {
-		return queue.AllActs()
+		ret = queue.AllActs()
+	}
+	addressAction, err := address.FromString(addr)
+	if err != nil {
+		return nil
+	}
+	for _, action := range ap.allActions {
```

Comment:
Can you leave a todo comment and link to the github issue for the performance concern?

Reply:
sure